### PR TITLE
JC-2291 List of User Groups

### PR DESCRIPTION
**Pull request by tk4nko**

### CODE REVIEW

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/GroupAdministrationDto.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    /**
+     * Instantiates new GroupDto object based on the Group object
+     *
+     * @param group source data for the DTO object
+     */
+    public GroupAdministrationDto(Group group) {
+        if (group != null) {
+            this.id = group.getId();
+            this.name = group.getName();
+            this.numberOfUsers = group.getUsers().size();
```
Comment (ctapobep) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/GroupAdministrationDto.java]: That would require to select all the users in the list unless the lazy=extra. We agreed that we don't want to select all the users, right?

File: jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/GroupAdministrationController.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+/**
+ * This controller handles actions with groups
+ * like display list of groups and number of users in it.
+ *
+ * @author Oleg Tkachenko
+ * @author Evgeniy Cheban
+ */
+@Controller
+public class GroupAdministrationController {
```
Comment (ctapobep) [inline jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/GroupAdministrationController.java]: Can we put this into other admin controller?

**Comment by ctapobep** [comment]
Could you please squash all these commits into one? I'll merge this to master afterwards.

**Comment by ctapobep** [comment]
I see that there are 12 of them..

**Comment by tk4nko** [comment]
Done and ready to merge.

**Pull request merged**